daemon/cmd/cni/cell.go

```diff
@@ -58,6 +63,7 @@ func (cfg Config) Flags(flags *pflag.FlagSet) {
 	flags.String(option.CNILogFile, defaultConfig.CNILogFile, "Path where the CNI plugin should write logs")
 	flags.String(option.CNIChainingTarget, defaultConfig.CNIChainingTarget, "CNI network name into which to insert the Cilium chained configuration. Use '*' to select any network.")
 	flags.Bool(option.CNIExclusive, defaultConfig.CNIExclusive, "Whether to remove other CNI configurations")
+	flags.Bool(option.CNIExternalRouting, defaultConfig.CNIExternalRouting, "Whether the other CNI handles routing on the node")
```

Comment:
nit: I would explicitly name _the other_ CNI as the `chained CNI plugin`. It's done the same in the comment of `ExternalRoutingEnabled() bool`

> Whether the other CNI handles routing on the node

->

> Whether the chained CNI plugin handles routing on the node

hint: don't forget to regenerate the CMD ref documentation (`make -C Documentation update-cmdref`)